app/grants/templates/grants/components/card.html

```diff
@@ -0,0 +1,232 @@
+                <span class="grant-item__owner-handle font-smaller-3">{% trans 'by' %}</span>
+                <a :href="grant.admin_profile.url" :data-usercard="grant.admin_profile.handle" target="_blank" rel="noopener noreferrer">
+                  <div class="d-flex align-center">
+                    <div class="grant-item__owner-image ml-2">
+                      <img :src="grant.admin_profile.avatar_url || 'https://c.gitcoin.co/avatars/57e79c0ae763bb27095f6b265a1a8bf3/thelostone-mc.svg'">
+                    </div>
+                    <div class="grant-item__owner-handle font-smaller-3">
+                      <span class="ml-2">
+                        [[ grant.admin_profile.handle || 'anonymous' ]]
+                      </span>
+                    </div>
+                  </div>
+                </a>
+              </div>
+              <div class="d-flex match ml-3">
+                <div class="match__round--inline mr-3">
+                  <template v-if="grant.is_clr_eligible">
+                    <span class="font-subheader font-weight-bold amount pt-1 mb-2">
+                      [[ get_clr_prediction(0, 1) || 0 ]] DAI
+                    </span>
+                    <span class="amount mb-2 font-smaller-3">
+                      Final CLR Match
+                    </span>
+                  </template>
+                  <template v-else>
+                    <span class="font-subheader font-weight-bold amount pt-1 mb-2">
+                      [[ grant.amount_received ]] DAI
+                    </span>
+                    <span class="amount mb-2 font-smaller-3">All Time Funding</span>
+                  </template>
+                </div>
+                <div>
+                  <span class="font-subheader font-weight-bold amount pt-1 mb-2">
+                    [[ grant.amount_received_in_round ]] DAI
+                  </span>
+                  <span class="amount mb-2 font-smaller-3">
+                    raised from
+                  </span>
+                  <span class="amount mb-2 font-smaller-3 font-weight-semibold">
+                   [[ grant.positive_round_contributor_count ]] contributors
+                  </span>
+                </div>
+              </div>
+              <template v-if="cred.is_authenticated">
+                <button :class="`ml-3 text-decoration-none btn btn-link star-action m-0 p-0 font-smaller-3 gc-text-blue ${!grant.favorite ? 'text-muted' : ''}`" :data-grant="grant.id">
+                  <i :class="`${!grant.favorite ? 'far' : 'fa' } fa-star`" ></i>
+                  <span class="ml-1 font-body">[[ !grant.favorite ? 'Follow' : 'Following' ]]</span>
+                </button>
+              </template>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-3 col-sm-12 py-4 pr-md-5 pl-md-0 grant-action">
+          <form class="js-addDetailToCart-form" v-if="!grant.is_on_team">
+            {% include 'grants/components/hidden_inputs.html' %}
```

Comment:
won't this be included multiple times @zoek1  ?

Reply:
yes, but removing this will require redoing the cart logic. I decided left to change in the future.